psi4/driver/driver.py

```diff
@@ -1931,37 +1931,121 @@ def gdma(wfn, datafile=""):
     if not datafile:
         os.remove(commands)
 
-
-def fchk(wfn, filename):
+def fchk(wfn, filename, debug=False):
```

Comment:
`NBOWriter` currently is a wavefunction method, but `fchk` and `molden` are top-level driver functions. Should we try to make these consistent, and if so, how? All we need for this PR is to decide if we want to make all of these wavefunction methods.

Reply:
I'm guessing we want to make them all wavefunction methods (attached to the class py-side if need be) as they're proliferating (fchk, molden, nbo, qcsk wfn). But `fchk(wfn)` and `molden(wfn)` don't get removed for years, if ever. No strong feelings, though.